lookout/style/format/feature_extractor.py

```diff
@@ -174,7 +213,7 @@ def extract_features(self, files: Iterable[File], lines: List[List[int]]=None
             return None
 
         y = numpy.concatenate(labels)
-        X = numpy.zeros((y.shape[0], self.count_features(FeatureGroup.all)), dtype=numpy.uint8)
+        X = numpy.zeros((y.shape[0], self.count_features()), dtype=numpy.uint8)
```

Comment:
Having FeatureGroup.all makes code unreadable in many places (every time you want to iterate over groups, you have to add a clause if feature_group is not FeatureGroup.all). So that's why I removed it.

Reply:
cool. I only for it.

Reply:
@vmarkovtsev I'll mark this as resolved but please ask for a new change if you'd rather have it the previous way (even though, as I stated before, I think it makes the code less readable).